packages/@magic-ext/connect/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@magic-ext/connect",
+  "version": "2.1.1",
```

Comment:
Is this version just copied over from oauth extension? Should we set it as a v1?

Reply:
Updated.